feature-libs/cart/saved-cart/components/add-to-saved-cart/add-to-saved-cart.component.ts

```diff
@@ -7,12 +7,8 @@ import {
   ViewChild,
   ViewContainerRef,
 } from '@angular/core';
-import {
-  ActiveCartService,
-  AuthService,
-  Cart,
-  RoutingService,
-} from '@spartacus/core';
+import { ActiveCartService } from '@spartacus/cart/main/core';
```

Comment:
Shouldn't we import the facade from cart/main/root instead?

Reply:
yes, but this class is that strange one, which make build fail.